pkg/upgrade/upgrade.go

```diff
@@ -319,6 +322,14 @@ func UpdateFromLegacyVersion(cli client.Client, platform deploy.Platform, appNS
 	return nil
 }
 
+// Special handling for cleanup dashboard jupyterhub CR
+func CleanupExistingResource(config *rest.Config, appNS string) error {
+	if err := RemoveUICR(config, gvr.JupyterhubApp, "jupyterhub", appNS); err != nil {
+		return err
+	}
+	return nil
```

Comment:
For now it can simply be:

```suggestion
	return RemoveUICR(config, gvr.JupyterhubApp, "jupyterhub", appNS)
```